docs/kubernetes/windows-details.md

```diff
@@ -0,0 +1,335 @@
+        "name": "windowspool2",
+        "count": 2,
+        "vmSize": "Standard_D2_v3",
+        "availabilityProfile": "AvailabilitySet",
+        "osType": "Windows",
+        "osDiskSizeGB": 128
+     }
+```
+
+### Choosing the Windows Server version
+
+If you want to deploy a specific Windows Server version, you can. First, find available versions with `az vm image list --publisher MicrosoftWindowsServer --all -o table`
+
+```
+$ az vm image list --publisher MicrosoftWindowsServer --all -o table
+
+Offer                    Publisher                      Sku                                             Urn                                                                                                            Version
+-----------------------  -----------------------------  ----------------------------------------------  -------------------------------------------------------------------------------------------------------------  -----------------
+...
+WindowsServerSemiAnnual  MicrosoftWindowsServer         Datacenter-Core-1709-with-Containers-smalldisk  MicrosoftWindowsServer:WindowsServerSemiAnnual:Datacenter-Core-1709-with-Containers-smalldisk:1709.0.20180412  1709.0.20180412
+WindowsServerSemiAnnual  MicrosoftWindowsServer         Datacenter-Core-1803-with-Containers-smalldisk  MicrosoftWindowsServer:WindowsServerSemiAnnual:Datacenter-Core-1803-with-Containers-smalldisk:1803.0.20180504  1803.0.20180504
+```
+
+You can use the Offer, Publisher and Sku to pick a specific version by adding `windowsOffer`, `windowsPublisher`, `windowsSku` and (optionally) `widndowsVersion` to the `windowsProfile` section. In this example, the latest Windows Server version 1803 image would be deployed.
+
+```json
+"windowsProfile": {
+            "adminUsername": "azureuser",
+            "adminPassword": "...",
+            "windowsPublisher": "MicrosoftWindowsServer",
+            "windowsOffer": "WindowsServerSemiAnnual",
+            "windowsSku": "Datacenter-Core-1803-with-Containers-smalldisk"
+     },
+```
+
+
+
+## More Examples
+
+### Using Azure Files
+
+This example is modified after https://github.com/andyzhangx/Demo/tree/master/windows/azurefile/rs3
+
+1. Create an azure file storage class
+```kubectl apply -f https://raw.githubusercontent.com/JiangtianLi/Examples/master/windows/azurefile/storageclass-azurefile.yaml```
+
+2. Make sure storageclass is created successfully
+```
+kubectl get storageclass/azurefile -o wide
+```
+
+3. Create a pvc for azure file
+```kubectl apply -f https://raw.githubusercontent.com/JiangtianLi/Examples/master/windows/azurefile/pvc-azurefile.yaml```
+
+4. Make sure pvc is created successfully
+```
+kubectl get pvc/pvc-azurefile -o wide
+```
+
+5. Create a pod with azure file pvc
+```kubectl apply -f https://raw.githubusercontent.com/JiangtianLi/Examples/master/windows/azurefile/iis-azurefile.yaml```
+
+6. Watch the status of pod until its `STATUS` is `Running`
+```
+watch kubectl get po/iis-azurefile -o wide

```

Comment:
I don't seem to have `watch` installed on macOS High Sierra. Maybe this command could use `kubectl -w` instead? I know its output gets muddled, but that would be more portable.

Reply:
Thanks! fixed in new changelist